frontend/src/components/header/Header.vue

```diff
@@ -31,6 +31,7 @@ const props = defineProps({
             <Button
                 class="btn-toggleSideNav"
                 @click="sideNavState.toggleSideNavVisible()"
+                aria-label="Toggle Side Nav"
```

Comment:
This was pointed out by the WAVE evaluation tool, because the toggle Icon did not have a descriptive text when active (mobile view) before.

Reply:
I think it is not a big deal, but wouldn't it be more descriptive in the screen reader if it was "toggle side navigation" instead of "toggle side navigation"?